src/client/css/user-menu.css

```diff
@@ -0,0 +1,109 @@
+.user-menu-wrapper {
+  font: normal 0.875rem/1.5 Inter, Inter-fallback, sans-serif;
+  outline: none;
+  position: relative;
+}
+
+.user-menu-button {
+  border: 2px solid transparent;
+  cursor: pointer;
+  display: block;
+  border-radius: 50%;
+  height: 42px;
+  overflow: hidden;
+  transition: border-color 150ms ease;
+  width: 42px;
+}
+
+.user-menu-button:hover,
+.user-menu-button:focus {
+  border-color: var(--purple-70);
+}
+
+.user-menu-list > hr {
+  border-image: var(--monitor-gradient) 1;
+  margin-bottom: var(--padding-sm);
+}
+
+.user-menu-popover {
+  background: white;
+  border-radius: 6px;
+  box-shadow: 0 2px 12px rgb(0 0 0 / 0.15);
+  list-style: none;
+  margin: var(--padding-md) 0 0 0;
+  min-width: var(--min-width);
+  max-width: var(--mobile-breakpoint);
+  padding: var(--padding-sm) 0;
+  position: absolute;
+  right: 0;
+}
+
+.user-menu-popover::after {
+  background-color: white;
+  border-top-left-radius: 1px;
+  content: '';
+  display: block;
+  height: 14px;
+  position: absolute;
+  right: calc(46px * 0.5);
+  top: 0;
+  transform: translate(50%, -50%) rotate(45deg);
+  width: 14px;
+}
+
+.user-menu-popover.visible {
+  opacity: 1;
+}
+
+.user-menu-header {
+  align-items: flex-start;
+  display: flex;
+  gap: var(--padding-xs);
+  flex-direction: column;
+  padding: var(--padding-md) calc(var(--padding-md) * 1.5);
+  position: relative;
+  text-decoration: none;
+}
+
+.user-menu-email {
+  color: var(--purple-90);
+  font: normal 0.875rem/1.5 Inter, Inter-fallback, sans-serif;
+  font-weight: 600;
+}
+
+.user-menu-fxa {
+  align-items: center;
+  display: flex;
+  gap: var(--padding-sm);
+}
+
+.user-menu-link,
+.user-menu-subtitle {
+  align-items: center;
+  color: var(--gray-50);
+  display: flex;
+  gap: var(--padding-sm);
+  padding: calc(var(--padding-sm) * 1.5) calc(var(--padding-md) * 1.5);
+  text-decoration: none;
+}
+
+.user-menu-subtitle {
+  padding: 0;
+}
+
+.user-menu-header:hover,
```

Comment:
Per discussion with Zachery, did we decide to remove the hover over the account header link to remain consistent with Relay?  I think removing the hover over the account link also makes more sense because it is more clearly an external link 🤷